packages/zoe/src/contracts/attestation/storedTime.js

```diff
@@ -0,0 +1,33 @@
+// @ts-check
+
+const { details: X, quote: q } = assert;
+
+/**
+ * @returns {{getTime: () => Timestamp, updateTime: (currentTime:
+ * Timestamp) => void}}
+ */
+const makeStoredTime = () => {
+  let storedTime = 0n;
+
+  const getTime = () => storedTime;
+
+  const updateTime = currentTime => {
+    assert.typeof(currentTime, 'bigint');
+    assert(
```

Comment:
Maybe this should not be an assertion?  This storedTime is contract-wide, so depending on the sequence and any delays in separate queries to the authority, might there be unexpected races?

Reply:
Sure, that is fine with me.